api/envoy/data/cluster/v2alpha/outlier_detection_event.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package envoy.data.cluster.v2alpha;
+option java_package = "io.envoyproxy.envoy.data.cluster.v2alpha";
+option java_multiple_files = true;
+
+import "google/protobuf/timestamp.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+option (gogoproto.equal_all) = true;
+
+// [#protodoc-title: Outlier detection logging events]
+// :ref:`Outlier detection logging <arch_overview_outlier_detection_logging>`.
+
+message OutlierDetectionEvent {
+  // In case of eject represents type of ejection that took place. Currently type can be one of
+  // ``5xx``, ``GatewayFailure`` or ``SuccessRate``
+  OutlierEjectionType type = 1 [(validate.rules).enum.defined_only = true];
+  // Timestamp for event.
+  google.protobuf.Timestamp timestamp = 2 [(gogoproto.stdtime) = true];
+  // The time in seconds since the last action (either an ejection or unejection) took place. This
+  // value will be ``-1`` for the first ejection given there is no action before the first ejection.
+  int64 secs_since_last_action = 3 [(validate.rules).int64.gte = 0];
```

Comment:
I don't think this validation is OK given that it can be -1. Related as long as we are here, thoughts on using the uint64t WKT (well known type) wrapper and just having it be absent if there is no last action? I think that might be more natural?